src/config.cpp

```diff
@@ -333,6 +334,8 @@ void parse_program_options(int argc, const char** argv)
     io_options.toggle("block-io",
                       "Enable recording of block I/O events (requires access to debugfs)");
 
+    nec_options.toggle("nec", "Enable NEC SX-Aurora Tsubasa sampling");
+    nec_options.option("nec-readout-interval").optional().metavar("MSEC").default_value("1");
```

Comment:
I don't think it's a good idea to have 1 millisecond as the default and minimal value. Why not read the interval as microseconds?